src/styles.css

```diff
@@ -8,22 +6,95 @@
   src: url('./assets/fonts/bayon.ttf') format('truetype');
 }
 
-:root, p {
-  --mdc-dialog-supporting-text-color: #f0edee;
-  --mdc-dialog-subhead-color: #f0edee;
+@font-face {
+  font-family: 'rubik-regular';
+  src: url('./assets/fonts/rubik/Rubik-Regular.ttf') format('truetype');
+}
+
+
+@font-face {
+  font-family: 'rubik-bold';
+  src: url('./assets/fonts/rubik/Rubik-SemiBold.ttf') format('truetype');
+}
+html, body {
+  height: 100%;
+  width: 100%;
 }
 
 body {
   background-color: #1e1e1e;
+  overflow: hidden;
```

Comment:
This should be removed because it doesn't allow me to scroll the page.

Reply:
I wonder if we could find a way to make some pages have custom overflow in css. On the other hand, temporary solution is setting overflow to 'auto' or 'scroll'.